cli_reference/developer_cli_odo/understanding-odo.adoc

```diff
@@ -40,46 +40,51 @@ In `{odo-title}`, services are provisioned from the OpenShift Service Catalog an
 [id="odo-supported-languages-and-images"]
 === Officially supported languages and corresponding container images
 
-.Supported languages, container images, and package managers
+.Supported languages, container images, package managers, and platforms
 [options="header"]
 |===
-|Language |Container image |Package manager
+|Language |Container image |Package manager |Platform
 |*Node.js*
-|https://registry.centos.org/centos/nodejs-10-centos7/latest[centos/nodejs-10-centos7]
-|NPM
-
-|
-|https://hub.docker.com/r/centos/nodejs-12-centos7[centos/nodejs-12-centos7]
+|https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/nodejs-10-rhel7[rhscl/nodejs-10-rhel7]
 |NPM
+|amd64, s390x
 
 |
 |https://catalog.redhat.com/software/containers/rhscl/nodejs-12-rhel7/[rhscl/nodejs-12-rhel7]
 |NPM
+|amd64, s390x
 
 |
-|https://www.github.com/bucharest-gold/centos7-s2i-nodejs[bucharestgold/centos7-s2i-nodejs]
+|https://access.redhat.com/articles/3376841[rhoar-nodejs/nodejs-10]
 |NPM
+|amd64
 
 |
-|https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/nodejs-8-rhel7[rhscl/nodejs-8-rhel7]
+|https://registry.centos.org/centos/nodejs-10-centos7/latest[centos/nodejs-10-centos7]
 |NPM
+|amd64
 
 |
-|https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/nodejs-10-rhel7[rhscl/nodejs-10-rhel7]
+|https://hub.docker.com/r/centos/nodejs-12-centos7[centos/nodejs-12-centos7]
 |NPM
+|amd64
 
 |*Java*
 |https://access.redhat.com/containers/#/registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift[redhat-openjdk-18/openjdk18-openshift]
 |Maven, Gradle
+|amd64, s390x
 
 |
 |https://access.redhat.com/containers/#/registry.access.redhat.com/openjdk/openjdk-11-rhel8[openjdk/openjdk-11-rhel8]
```

Comment:
It looks that https://access.redhat.com/containers/#/registry.access.redhat.com/openjdk/openjdk-11-rhel8 is now Deprecated, will you use the new one "https://access.redhat.com/containers/#/registry.access.redhat.com/ubi8/openjdk-11/"

Reply:
@boczkowska Same here.

Reply:
odo does not support `ubi8` image currently.  For questions and discussions about it, file an issue at https://github.com/openshift/odo/pulls or join the openshiftdo.slack.com

Reply:
>Will you use "https://hub.docker.com/r/centos/nodejs-10-centos7" to replace "https://registry.centos.org/centos/nodejs-10-centos7/latest"

Not that I'm aware of.